[INFO     2010-08-31 13:54:08,959 bulkloader.py] Logging to bulkloader-log-20100831.135408
[INFO     2010-08-31 13:54:08,959 bulkloader.py] Throttling transfers:
[INFO     2010-08-31 13:54:08,959 bulkloader.py] Bandwidth: 250000 bytes/second
[INFO     2010-08-31 13:54:08,959 bulkloader.py] HTTP connections: 8/second
[INFO     2010-08-31 13:54:08,959 bulkloader.py] Entities inserted/fetched/modified: 20/second
[INFO     2010-08-31 13:54:08,959 bulkloader.py] Batch Size: 10
[INFO     2010-08-31 13:54:08,961 bulkloader.py] Opening database: bulkloader-progress-20100831.135408.sql3
[DEBUG    2010-08-31 13:54:09,014 adaptive_thread_pool.py] [Thread-1] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,024 adaptive_thread_pool.py] [Thread-2] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,037 adaptive_thread_pool.py] [Thread-3] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,047 adaptive_thread_pool.py] [Thread-4] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,057 adaptive_thread_pool.py] [Thread-5] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,068 adaptive_thread_pool.py] [Thread-6] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,078 adaptive_thread_pool.py] [Thread-7] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,088 adaptive_thread_pool.py] [Thread-8] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,099 adaptive_thread_pool.py] [Thread-9] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,109 adaptive_thread_pool.py] [Thread-10] WorkerThread: started
[DEBUG    2010-08-31 13:54:09,119 bulkloader.py] Configuring remote_api. url_path = /remote_api, servername = mk10koers.appspot.com
[DEBUG    2010-08-31 13:54:09,173 bulkloader.py] Bulkloader using app_id: mk10koers
[INFO     2010-08-31 13:54:09,174 bulkloader.py] Connecting to mk10koers.appspot.com/remote_api
[DEBUG    2010-08-31 13:54:09,470 bulkloader.py] [Thread-11] ProgressTrackerThread: started
[DEBUG    2010-08-31 13:54:09,481 bulkloader.py] [Thread-12] DataSourceThread: started
[INFO     2010-08-31 13:54:09,481 bulkloader.py] Starting import; maximum 10 entities per post
[DEBUG    2010-08-31 13:54:09,502 bulkloader.py] [Thread-12] DataSourceThread: exiting
[DEBUG    2010-08-31 13:54:09,511 bulkloader.py] Waiting for worker threads to finish...
[DEBUG    2010-08-31 13:54:09,522 adaptive_thread_pool.py] [Thread-1] Got work item [1-10]
[ERROR    2010-08-31 13:54:09,533 adaptive_thread_pool.py] [Thread-1] WorkerThread:
Traceback (most recent call last):
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/adaptive_thread_pool.py", line 150, in WorkOnItems
    status, instruction = item.PerformWork(self.__thread_pool)
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/bulkloader.py", line 691, in PerformWork
    transfer_time = self._TransferItem(thread_pool)
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/bulkloader.py", line 846, in _TransferItem
    self.content = self.request_manager.EncodeContent(self.rows)
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/bulkloader.py", line 1267, in EncodeContent
    entity = loader.create_entity(values, key_name=key, parent=parent)
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/bulkloader.py", line 2522, in create_entity
    properties[name] = converter(val)
ValueError: invalid literal for int() with base 10: 'Lengte in cm'
[DEBUG    2010-08-31 13:54:09,536 adaptive_thread_pool.py] [Thread-1] WorkerThread: exiting
[INFO     2010-08-31 13:54:09,536 adaptive_thread_pool.py] [Thread-2] Backing off due to errors: 1.0 seconds
[INFO     2010-08-31 13:54:10,023 bulkloader.py] An error occurred. Shutting down...
[DEBUG    2010-08-31 13:54:10,024 adaptive_thread_pool.py] [Thread-4] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,024 adaptive_thread_pool.py] [Thread-5] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,025 adaptive_thread_pool.py] [Thread-3] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,025 adaptive_thread_pool.py] [Thread-7] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,026 adaptive_thread_pool.py] [Thread-10] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,026 adaptive_thread_pool.py] [Thread-6] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,026 adaptive_thread_pool.py] [Thread-8] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,026 adaptive_thread_pool.py] [Thread-9] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,028 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,029 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,037 adaptive_thread_pool.py] [Thread-2] WorkerThread: exiting
[DEBUG    2010-08-31 13:54:10,037 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,038 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,038 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,038 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,038 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,038 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,038 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[DEBUG    2010-08-31 13:54:10,038 adaptive_thread_pool.py] Waiting for Anonymous_WorkerThread to exit
[ERROR    2010-08-31 13:54:10,039 adaptive_thread_pool.py] Error in Thread-1: invalid literal for int() with base 10: 'Lengte in cm'
[DEBUG    2010-08-31 13:54:10,039 adaptive_thread_pool.py] Traceback (most recent call last):
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/adaptive_thread_pool.py", line 150, in WorkOnItems
    status, instruction = item.PerformWork(self.__thread_pool)
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/bulkloader.py", line 691, in PerformWork
    transfer_time = self._TransferItem(thread_pool)
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/bulkloader.py", line 846, in _TransferItem
    self.content = self.request_manager.EncodeContent(self.rows)
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/bulkloader.py", line 1267, in EncodeContent
    entity = loader.create_entity(values, key_name=key, parent=parent)
  File "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine/google/appengine/tools/bulkloader.py", line 2522, in create_entity
    properties[name] = converter(val)
ValueError: invalid literal for int() with base 10: 'Lengte in cm'

[DEBUG    2010-08-31 13:54:10,050 bulkloader.py] Waiting for progress_thread to terminate...
[DEBUG    2010-08-31 13:54:10,117 bulkloader.py] [Thread-11] ProgressTrackerThread: exiting
[DEBUG    2010-08-31 13:54:10,139 bulkloader.py] ... done.
[INFO     2010-08-31 13:54:10,139 bulkloader.py] 13 entites total, 0 previously transferred
[INFO     2010-08-31 13:54:10,140 bulkloader.py] 0 entities (724 bytes) transferred in 1.2 seconds
[INFO     2010-08-31 13:54:10,140 bulkloader.py] Some entities not successfully transferred
